inst/assets/rmarkdown_template.html

```diff
@@ -0,0 +1,107 @@
+<html>
+  <head>
+    <title>$title$</title>
+    <style>
+      :root {
+        --black-color: black;
+        --main-font: "Calibri", sans-serif;
+        --base-font-size: 18px;
+      }
+
+      body {
+        font-family: var(--main-font);
+        font-size: var(--base-font-size);
+        margin: 0;
+        padding: 0;
+      }
+
+      h1,
+      h2,
+      h3 {
+        font-family: "Arial", sans-serif;
+        font-weight: bold;
+        color: var(--black-color);
+      }
+
+      h1 {
+        font-size: calc(15pt * 2);
+      }
+
+      h2 {
+        font-size: calc(14pt * 2);
+      }
+
+      h3 {
+        font-size: calc(13pt * 2);
+      }
+
+      #header {
+        background-color: #314160;
+        width: 100%;
+        padding: 40px 0;
+        position: relative;
+        border: 0;
+      }
+
+      .header-wrapper {
+        max-width: 940px;
+        margin: 0 auto;
+        padding: 0 20px;
+      }
+
+      .header-content {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        margin-bottom: 15px;
+      }
+
+      .logo {
+        height: 40px;
+        width: auto;
+        background-image: url(logo/OMNI_Logo-White_Horizontal.png);
```

Comment:
@tvroylandt do you know how the path is supposed to looks like? I looked at other examples in the package and none of them seem to work. Is it supposed to be relative to the template?

Reply:
See https://github.com/rfortherestofus/omni/blob/fix-assets-in-html/inst/rmarkdown/templates/html_rmarkdown/skeleton/template.html

Reply:
thanks!